arc-core/src/arc/input/KeyCode.java

```diff
@@ -37,6 +37,7 @@
     //scroll
     scroll(KeyType.scroll, "Scrollwheel", true),
     //keyboard
+    nullKey(KeyType.key, "Empty"),
```

Comment:
There already is an 'unset' key.

Reply:
But the unset key reverts it back to the default it seems.